docs/hooks/useStepper.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Canvas, Meta } from "@storybook/addon-docs";
+import { UseStepper } from "./UseStepper";
+
+<Meta title="Hooks/useStepper" />
+
+# UseStepper
+
+The `useStepper` hook provides a type-safe way to manage multi-step flows in
+React applications. It handles step progression and navigation while ensuring
+type safety for step definitions.
+
```

Comment:
Suggest adding a code snippet for how to import. (We're not super consistent on this but it is helpful.) Github messes with the formatting here, but I think you get the idea:
```suggestion
```typescript
import { useStepper } from "@jobber/hooks/useStepper";
````
```